1-mandatory/2-people-i-know.js

```diff
@@ -378,14 +378,22 @@ First, I want you to find all of my friends who are 35 or older.
 */
 
```

Comment:
Good job on all of these. Now that you're comfortable with `for` loops, I would definitely recommend you try them with array methods.